kikit/panelize_ui.py

```diff
@@ -34,6 +35,44 @@ def splitStr(delimiter, escapeChar, s):
         return x
 
 
+class Input(click.ParamType):
+    """
+    A CLI argument type for defining input boards. Can be either
+    - Path to PCB file
+    - Path to JSON file containing list of Board objects
+    - JSON string containing list of Board objects
+    """
+    name = "input"
+
+    def convert(self, value, param, ctx):
+        if len(value.strip()) == 0:
+            self.fail(f"{value} is not a valid argument specification",
+                param, ctx)
+        try:
+            values = []
+            if value.endswith(".kicad_pcb"):
+                values.append(Board(value, ["0mm","0mm"]))
+            else:
+                json_str = value
+                s = Section()
```

Comment:
Using `Section` object in order to not rewrite the conversion code. 
Might be cleaner to move that code out into it's own function?